src/CMake/nativeTests.cmake

```diff
@@ -12,11 +12,18 @@ add_test(NAME xbutil
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
 
 add_test(NAME xbutil2
-  COMMAND ${CMAKE_BINARY_DIR}/runtime_src/core/tools/xbutil2/xbutil2 --new scan
+  COMMAND ${CMAKE_BINARY_DIR}/runtime_src/core/tools/xbutil2/xbutil2 --new examine
```

Comment:
we don't need to specify '--new' with xbutil**2**

Reply:
Good catch! Updated.